### ISSUE TITLE
Mod Config not showing up

### ISSUE THREAD

**Issue by Jboy9786** [Neanka/MCM_0.1_AS3#86]
I already have F4SE and launched the game through f4se loader, does it have something to do with the load order ?

**Comment by Xariss00** [comment]
Copy/Paste of a relevant message from the MCM/XDI Discord:

> Well, that's a bit vague.
> 
> I'm all for helping folks get MCM working.  It's an amazingly powerful addition to any mod collection and I believe everyone deserves to have access to its awesomeness.  Unfortunately, I am not omniscient; I am a newb and an idiot.  I need at least a little information to work with beyond "it don't work."  😄  So, any chance you could provide just a teensy bit more info?  For instance:
> 
> What platform are you playing on (Steam, GOG, Epic, etc.)?
> 
> Do you use a mod organizer or install mods manually?  If an organizer, which one?
> 
> Are you aware that MCM only appears in the Pause menu, not the main menu?
> 
> Open the game.  At the main menu, click on Settings.  In the lower right corner, which versions of the game and F4SE does it say are actually being loaded? (may not match what you think is installed)
> 
> Which version of MCM do you have installed? (do NOT say "the latest one")

**Comment by GhastlyWolf2392** [comment]
I'm having the same problem 😢. The f4se is 7.2 version and the the mcm is 1.39 version.

**Comment by tom-tom80** [comment]
i am using steam and mod organizer and at the bottom on the log section it says [img align=right]https://i.imgur.com/yThthRV.png[/img] doesn't match the expression for img

**Comment by Swiftkill** [comment]
FO4 1.10.984 F4SE 0.7.2, No version of MCM, 1.40 included appear or work,any related functionality in other mods also get "sunk"(or alternate route, e.g. LSTRING Bow ammo switcher), as if nothing happens at all.

**Comment by Fadamor** [comment]
Mod Configuration Menu is installed and enabled by way of Vortex. I do not get the Mod Config menu item on-screen when I hit ESC after a save file is loaded.
Here's everything I can provide about the issue on my computer:
Currently using:
FO4 NG for PC (Steam Version)
F4SE version ------------ **EDIT: Version 0.7.2**
Mod Configuration Menu 1.40 BETA 2 Version 1.40 BETA2 - INSTALLED and ENABLED in Vortex
MCM Settings Manager Version 1.2 - INSTALLED, but DISABLED in Vortex
MCM Booster Version 1.0 - INSTALLED, but DISABLED in Vortex
MCM Booster NG V1.0 Version 1.0 - INSTALLED, but DISABLED in Vortex

There are 918 mods installed in total, of which 241 mods are enabled (including the ones listed above) in Vortex. Most of the enabled mods relate to Sim Settlements 2 or bug fixes/patches

Any suggestions on how to get the Mod Config menu item to properly appear?

**Comment by Xariss00** [comment]
Running next-gen, your F4SE version should be v0.7.2. If Vortex is really showing it as 1.10.984.0.1, that would be my first guess - kill your existing F4SE and grab a fresh one.

I know it's sounds silly, but it's worth a shot.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Fadamor** [comment]
Hmmm, There was a 0.7.?  in the settings menu. Let me check it again...

In the Settings menu, it shows:
"v (F4SE 7.4.2) 1.10.984.0.1"

EDIT: I'm an idiot. It actually says "v (F4SE 0.7.2) 1.10.984.0.1"

**Comment by Fadamor** [comment]
I'm guessing, then, that the "1.10.984.0.1" is the version number for FO4 NG?

**Comment by Xariss00** [comment]
Never seen/ heard of a version like that. Here is a link to the correct version, v0.7.2.

https://www.nexusmods.com/fallout4/mods/42147

________________________________

**Comment by Fadamor** [comment]
> Never seen/ heard of a version like that. Here is a link to the correct version, v0.7.2.
Go into FO4, open Settings, and in small print to the right you will see:

<img width="241" height="98" alt="Image" src="https://github.com/user-attachments/assets/bcda45fe-c73f-4f2d-809d-672b7fe42287" />

Now that I've had a good sleep, it seems obvious the stuff within the parenthesis relates to F4SE and the stuff after the parenthesis relates to the Fallout 4 version.

**Comment by Xariss00** [comment]
Correct. Now that e know your versions are correct, however, that means the problem is somewhere else. 

The only other thing I could recommend would be to completely remove MCM and the boosters, then try a manual install of MCM.

**Comment by Fadamor** [comment]
I'll give it a try. I'm completely stumped on this end.

**Comment by Fadamor** [comment]
After a manual install, I now have MCM working - at least to the point the menu option is there. Kudos to [Xariss00](https://github.com/Xariss00) for his patience and an absolute "Thank You" for the assistance. 

For anyone who wants to try my steps, they follow:
 
1. Completely uninstalled any MCM mods using the Vortex "Uninstall" function - including deleting archives.
2. Manually downloaded "Mod Configuration Menu 1.40 BETA 2-21497-1-40-BETA2-1715646081.zip" from NexusMods.com and extracted the contents to its own sub-folder under "Downloads"
3. Copied the resulting "Data" folder and pasted it into the "E:\SteamLibrary\steamapps\common\Fallout 4" directory (use the file path to your own Fallout 4 installation folder)
4. Ran Fallout 4 and loaded the most recent save file.
5. Hit ESC to bring up the Pause Screen.
6. Verified the presence of "MOD CONFIG"" at the top of the list.
